### PR TITLE
Added spec for prescriptions

### LINKED ISSUE #24: Prescriptions

### PR DESCRIPTION
**Pull request by jarinykanen**
Resolves #24

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -606,6 +607,67 @@ paths:
...
 
+  /v1/prescriptions:
+    get:
+      summary: Get patients prescriptions
+      description: Gets patients prescriptions
+      tags:
+        - Prescriptions
+      operationId: listPrescriptions
+      parameters:
+        - name: count
```
Comment (anttileppa) [inline swagger.yaml:618]: we have usually used "maxResults" and firstResult parameters

File: swagger.yaml [hunk 2]
```diff
@@ -606,6 +607,67 @@ paths:
...
+          schema:
+            type: integer
+            format: int32
+        - name: startTime
+          in: query
+          required: true
+          description: From which date onwards the search is made
+          schema:
+            type: string
+            format: date-time
```
Comment (anttileppa) [inline swagger.yaml:631]: is it really date-time or just date?

File: swagger.yaml [hunk 3]
```diff
@@ -606,6 +607,67 @@ paths:
...
+        - name: endTime
+          in: query
+          required: true
+          description: Until which date the search is made
+          schema:
+            type: string
+            format: date-time
```
Comment (anttileppa) [inline swagger.yaml:638]: is it really date-time or just date?

File: swagger.yaml [hunk 4]
```diff
@@ -606,6 +607,67 @@ paths:
...
+          in: query
+          required: true
+          description: From which date onwards the search is made
+          schema:
+            type: string
+            format: date-time
+        - name: endTime
+          in: query
+          required: true
+          description: Until which date the search is made
```
Comment (anttileppa) [inline swagger.yaml:635]: Add description whether it includes or excludes given date

File: swagger.yaml [hunk 5]
```diff
@@ -606,6 +607,67 @@ paths:
...
+          in: query
+          required: false
+          description: If this is given only the given amount of latest prescriptions are retrieved
+          schema:
+            type: integer
+            format: int32
+        - name: startTime
+          in: query
+          required: true
+          description: From which date onwards the search is made
```
Comment (anttileppa) [inline swagger.yaml:628]: Add description whether it includes or excludes given date

File: swagger.yaml [hunk 6]
```diff
@@ -827,3 +889,29 @@ components:
...
+
+    Prescription:
+      type: object
+      description: Prescription object
+      required:
+        - prescriptionDate
+        - prescriber
+        - description
+        - validUntil
+      properties:
```
Comment (anttileppa) [inline swagger.yaml:901]: It does not have any kind of id?

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: swagger.yaml (hunk 6 above)
Comment (jarinykanen) [reply swagger.yaml:901]: Nope

Comment (anttileppa) [reply swagger.yaml:901]: And I suppose one cannot be deducted from other details?

**Review by anttileppa — APPROVED** [review]

**Pull request merged**